src/components/EditorWidgets/Date/DateControl.js

```diff
@@ -42,7 +42,7 @@ export default class DateControl extends React.Component {
 
   render() {
     const { includeTime, value, classNameWrapper, setActiveStyle, setInactiveStyle } = this.props;
-    const format = this.format || moment.defaultFormat;
+    const format = this.format || moment[includeTime ? 'defaultFormat' : 'HTML5_FMT.DATE'];
```

Comment:
`'HTML5_FMT.DATE'` won't work as it's not a key name, since `DATE` is a property of `HTML5_FMT`.

Reply:
```js
const format = this.format || (includeTime ? moment.defaultFormat : moment.HTML5_FMT.DATE);
```
should work (not tested)

Reply:
Good catch!